Assets/Scripts/Gameplay/Tile.cs

```diff
@@ -31,6 +31,9 @@ public enum Anchor
         
         private List<Structure> m_structures;
 
+        // Public viewable list of structures. TODO: The structures in the list are still editable.
+        public List<Structure> Structures => new List<Structure>(m_structures);
```

Comment:
You can still change the internal structure, so not sure what the point of wrapping this in another list.

Reply:
Made m_structures public to return it instead